velox/dwio/common/DirectBufferedInput.cpp

```diff
@@ -274,6 +274,7 @@ std::vector<cache::CachePin> DirectCoalescedLoad::loadData(bool isPrefetch) {
   }
   input_->read(buffers, requests_[0].region.offset, LogType::FILE);
   ioStats_->read().increment(size);
+  ioStats_->incRawBytesRead(size);
```

Comment:
Looks it's a duplicated metric. Can we use  ioStats_->Read instead?

Reply:
@FelixYBW Yes, the duplicated metric has been removed. Additionally, the `getCompletedBytes `method in `HiveDataSource `has been updated to retrieve the` ioStats_->read()` metric instead of the `rawBytesRead_ `metric. Here is a detailed explanation of the changes https://github.com/facebookincubator/velox/pull/8147#issue-2052161962

Reply:
@JkSelf do we want to increment incRawBytesRead here? From my discussion with @oerling, rawBytesRead counts the number of bytes read by query which shall only be incremented in loadSync? Thanks!

Reply:
@xiaoxmeng Got it. Updated. Please help to review again. Thanks.